ivys.js

```diff
@@ -26,14 +26,55 @@ IvyApp.prototype.removeItem = function() {
     
 };
 
-IvyApp.prototype.addTag = function() {
-};
-
-IvyApp.prototype.removeTag = function() {
-};
-
 // Footer menu arrow function 
 IvyApp.prototype.toggleFooterMenu = function() {
     // Toggles footer menu
     $(this).parent().parent().children('.menu-footer').collapse('toggle');
 }
+
+
+// Product State and Product Class skeletons
+
+var emptyProductState = function(params) {
+    
+    return {
+        // seaWinds Id, SKU number?
+        id: params.seaWindsId,
+        // Product name
+        name: params.name,
+        // product tags
+        tags: [],
```

Comment:
I mean, I just want to be able to add a tag, remove a tag, maybe search for specific tags? So, should I have tags: {} instead? So that I can find specific tags?